cmd/convert.go

```diff
@@ -20,7 +21,11 @@ var convertCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal(err)
 		}
-		err = markdownService.ConvertToMarkdown(file, outputDir)
+		if titleFormat == "MethodURL" {
+			err = markdownService.ConvertToMarkdown(file, outputDir, true)
+		} else {
+			err = markdownService.ConvertToMarkdown(file, outputDir, false)
+		}
```

Comment:
```suggestion
                var titleFormatSetting = false
                if titleFormat == "MethodURL" {
                  titleFormatSetting = true
                }
	   err = markdownService.ConvertToMarkdown(file, outputDir, titleFormatSetting)
```

Reply:
So i have been giving your suggestion some thought, is there a reason you prefer it other than that I pass true/false into the method? Because I usually would avoid initialising a new variable when I am only using it once as there is less to garbage collect? If it is SPAN convention/you have strong feelings about it I am happy to submit it, there was actually a type in my code I only just found. I last min change MethodURL to methodURL and forgot to change it here. So I will fix that regardless.

Reply:
On further discussion, I am not going to change the above. Rather I will just fix the one type.
```suggestion
		if titleFormat == "methodURL" {
```